pkg/graph/resolvers/it_gov_task_statuses.go

```diff
@@ -99,8 +99,30 @@ func BizCaseFinalStatus(intake *models.SystemIntake) models.ITGovFinalBusinessCa
 }
 
 // GrbMeetingStatus calculates the ITGovGRBStatus for the GrbMeeting section for the system intake task list for the requester view
-func GrbMeetingStatus(intake *models.SystemIntake) models.ITGovGRBStatus {
-	return models.ITGGRBSCantStart
+func GrbMeetingStatus(intake *models.SystemIntake) (models.ITGovGRBStatus, error) {
+	if intake.GRBDate != nil { // status depends on if there is a date scheduled or not
+		if intake.GRBDate.After(time.Now()) { // Meeting has not happened
+			return models.ITGGRBSScheduled, nil
+		}
+		if intake.Step == models.SystemIntakeStepGRBMEETING { //if the step is GRB meeting, status is awaiting decision
+			return models.ITGGRBSAwaitingDecision, nil
+		}
+		return models.ITGGRBSCompleted, nil // if the step is not GRB meeting, the status is completed
+	}
+	// the grb date is not nil.
```

Comment:
Should be "is nil" right?

Reply:
Thanks Lee, you are right! Correcting now.